UpstreamTagger/UpstreamTagger.cxx

```diff
@@ -379,7 +379,7 @@ void UpstreamTagger::ConstructGeometry()
 
   InitMedium("vacuum");  
   TGeoMedium *Vacuum_box =gGeoManager->GetMedium("vacuum");
-  double xbox=233.4; double ybox = 113; double zbox = 1.3503;
```

Comment:
@yabezsh Did you ever understand how these numbers are calculated?

Reply:
I do not remember exactly what these values are, but if you plot them, it becomes clear. Essentially, they are different blocks, and you can tell by their position that they should be shorter, etc. I have no idea if the numbers are from real size blocks. We need someone who is familiar with UT

Reply:
For now I scaled the the numbers naively for #420, and the result looks more or less OK. Let's see what happens once we check for overlaps...
Scaling of the other parts seems more difficult right now. Things didn't quite scale as I expected.